Circular_kernel_3/demo/Circular_kernel_3/Circular_kernel_3.cpp

```diff
@@ -3,15 +3,25 @@
 
 int main(int argc, char** argv)
 {
+  QSurfaceFormat fmt;
```

Comment:
Same question: why here you don't use code_to_call_before_creation_of_QCoreApplication ?